src/panels/logbook/ha-logbook.ts

```diff
@@ -254,6 +265,126 @@ class HaLogbook extends LitElement {
     this._savedScrollPos = (e.target as HTMLDivElement).scrollTop;
   }
 
+  private _formatEventBy = (item: LogbookEntry, seenEntities: string[]) => {
```

Comment:
This method is not used as an event handler so doesn't need to be defined as an arrow function.
```suggestion
  private _formatEventBy(item: LogbookEntry, seenEntities: string[]) {
```